src/tests/unit/tests/electron/platform/android/setup/live-android-setup-deps.test.ts

```diff
@@ -37,12 +41,18 @@ describe('LiveAndroidSetupDeps', () => {
         );
         fetchConfigMock = Mock.ofInstance((port: number) => new Promise<DeviceConfig>(() => null));
         loggerMock = Mock.ofType<Logger>();
+        adbWrapperFactoryMock = Mock.ofType<AdbWrapperFactory>();
+        adbWrapperHolderMock = Mock.ofType<AdbWrapperHolder>();
+        adbWrapperStub = {} as AdbWrapper;
```

Comment:
My inclination would be to also use a strict mock for adbWrapperStub, but that may just be my C# reflexes kicking in...